src/pages/guidelines/themes/code.mdx

```diff
@@ -87,7 +94,7 @@ Alternatively, you can set the global theme variable then call the mixin without
 passing in a theme name.
 
 ```scss
-@import '@carbon/themes/scss/themes';
+@use '@carbon/themes/scss/themes';
 
 $theme: $theme--g10;
```

Comment:
This usage is outdated, right?

Reply:
Is it? I thought that's how you accessed the theming tokens, but I'd bet i'm definitely wrong

Reply:
i thought that was how we still brought in themes for custom work? but I could just be doing it wrong